soundcard/mediafoundation.py

```diff
@@ -15,7 +15,10 @@
 _combase = _ffi.dlopen('combase')
 _ole32 = _ffi.dlopen('ole32')
 
+
+
 class _COMLibrary:
+    comLoadedByThisClass = False;
```

Comment:
Why is this necessary as a class variable? Isn't it already set in `__init__`?

Reply:
Tested and it is indeed not necessary. Removed in new PR.
I didn't know that this definition is not required because I am relatively new to Python